axisregistry/terminal_slope.textproto

```diff
@@ -0,0 +1,19 @@
+#SHARP based on https://github.com/monokromskriftforlag/geologisk/#variable-axis
+tag: "TSLO"
+display_name: "Terminal_Slope"
+min_value: 0.0
+default_value: 0.0
+max_value: 100.0
+precision: 1
+fallback {
+  name: "Flat"
+  value: 0.0
+}
+fallback {
+  name: "Sloped"
+  value: 100.0
+}
+description: "Detailing and terminal treatment."
+  "Modifies the slope of the sans serif stem terminals from flat to sloped"
+  "As the slop increase, different levels of acuteness can be set"
```

Comment:
Perhaps the description could simply be "Description: Adjust the slope angle of sans-serif stem terminals."?

Reply:
Done